.github/workflows/lighthouse-ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         id: netlify
         with:
           site_name: asyncapi-website
-          max_timeout: 300
+          max_timeout: 600
```

Comment:
instead of 5 we need to wait 10min for preview build as it is most of the time 5+ as Lighthouse check fails quite often and requires rerun